articles/getting-started-with-browser-sync/index.md

```diff
@@ -0,0 +1,62 @@
+# Getting Started with Browser-sync
+
+
+Browser-sync is an automation tool that makes your web development project faster. It makes your browser testing workflow faster by synchronising URLs, interactions and code changes across multiple devices, both internally and externally.
+
+*I have been working with different local servers just to replace the hasle of live servers for your web app or web design testing but I got stucked with [browser-sync](https://www.browsersync.io/) because of it’s complete package. You get a time-saving synchronised browser testing.* 
```

Comment:
spelling error = hassle. I got stucked doesn't make sense. 

You should spell-check your article and review for grammar and readability before submitting. 

Use Grammarly (linked in checklist) and read it back to yourself like you're reading it aloud. Does the article flow well?